collector/lib/HostHeuristics.cpp

```diff
@@ -23,9 +23,15 @@ class CollectionHeuristic : public Heuristic {
                   << " does not support eBPF, which is a requirement for Collector.";
     }
 
+    auto kernel = host.GetKernelVersion();
     // If we're configured to use eBPF with BTF, we try to be conservative
     // and fail instead of falling-back to ebpf.
     if (config.GetCollectionMethod() == CollectionMethod::CORE_BPF) {
+      if (kernel.machine == "ppc64le") {
+        CLOG(FATAL) << "CORE_BPF collection method is not supported on ppc64le. "
+                    << "Please set collector.collectionMethod=EBPF.";
```

Comment:
Maybe reword it for consistency with other similar messages?
```
"HINT: You may alternatively want to use eBPF based collection "
"with collector.collectionMethod=EBPF."
```

Reply:
My only objection to this is the proposed message makes it seem like this is an option, when in reality collector will not work with `CORE_BPF`. WDYT about:
```cpp
"HINT: Change collection method to EBPF with collector.collectionMethod=EBPF."
```

Reply:
Why does it sound like an option to you? It clearly says, CORE_BPF is not supported here, but you could use some other collection method. The other cases down below are the same, something is not supported, and the error message says exactly that.

Reply:
Because `You may alternatively want` is a very passive way of stating something you HAVE to do.

Reply:
> The other cases down below are the same, something is not supported, and the error message says exactly that.

I also don't agree with the other message being so passive, but I'm not changing those in this PR.

Reply:
Changed wording on 2 of the messages, the one that is left with `You may alternatively want` makes sense to keep that wording because providing BTF symbols would also be an option.